docs/custom-rules.md

```diff
@@ -21,6 +21,26 @@ linter rules might then look something like this:
 If you so prefer, custom rules may also be provided using the `--rules` option for `regal lint`, which may point either
 to a Rego file, or a directory containing Rego files and potentially data (JSON or YAML).
 
+## Creating a New Rule
```

Comment:
Is it worth calling out this functionality for creating built-in rules in the readme so to guide new contributors?

Reply:
Hmm, I guess that'd be in the development.md file? I'll amend this with a note there.